src/Controller/OcrController.php

```diff
@@ -58,7 +58,7 @@ public function __construct(RequestStack $requestStack, Intuition $intuition, En
         // Parameters.
         $this->imageUrl = (string)$request->query->get('image');
         static::$params['langs'] = $this->getLangs($request);
-        static::$params['image_hosts'] = $this->intuition->listToText($this->engine->getImageHosts());
+        static::$params['image_hosts'] = $this->engine->getImageHosts();
```

Comment:
I changed this so that the API endpoint returns an array instead of a string. In the error message it still uses Intuition::listToText()